buildcfg.txt

```diff
@@ -2173,10 +2173,10 @@ BKEYB0	7	19
 BLUDA0	8	5
 BLUDB0	6	7
 BLUDC0	3	5
-BON1A0	5	21
-BON1B0	5	21
-BON1C0	6	21
-BON1D0	6	21
+BON1A0	5	15
+BON1B0	5	15
+BON1C0	5	15
+BON1D0	5	15
```

Comment:
The image sizes have changed from 11x22 to 11x15. Since the image coordinates are zero based previously the offset, or foot of the image, was a pixel on the bottom row inside the image. Now it's just below the bottom of the image. Inside the image feels more correct to me, but I don't know that it matters since it seems to look fine the way it is with this change.

Reply:
The idea of having the offset be a little bit up from the bottom of the sprite is to give the illusion of depth, like the foot of the thing is extending forwards a bit.

It starts looking a lot less consistent in sourceports that let you look up and down; in some hardware accelerated implementations (e.g., GZDoom) the bottom part just clips into the floor like it's sunken in.

This particular sprite isn't drawn with that perspective trick in the first place, though, so the offset being at the very bottom is correct.